docs/Superoperator representations.md

```diff
@@ -12,49 +16,565 @@ where $i$ is a row and $j$ is a column index.
 
 We define `vec` to be column stacking
 
-$ {\rm vec}(A) = (a_{11},a_{21},\ldots,a_{m1},a_{12},\ldots,a_{mm})^T$
+$$ |A\rangle \rangle :={\rm vec}(A) = (a_{11},a_{21},\ldots,a_{m1},a_{12},\ldots,a_{mm})^T \quad (1) $$
 
 were $T$ denotes a transpose. Clearly an inverse operation, `unvec` can be defined so that 
 
-$ {\rm unvec}\big ( {\rm vec}(A) \big ) = A$.
+$$ {\rm unvec}\big ( {\rm vec}(A) \big ) = A.$$
+
+Similarly we can define a row vectorization to be row stacking $ {\rm vec_r}(A) = (a_{11}, a_{12}, \ldots, a_{1m}, a_{21},\ldots, a_{mm})^T$. Note that ${\rm vec}(A) = {\rm vec_r}(A^T)$. In any case we will **not** use this row convention.
```

Comment:
This is actually an interesting point.  In Python it may be more efficient to do row-major ordering.  The reason is that, if I remember correctly, and @stylewarning or @blakejohnson or @caryan can correct me, Python uses row-major conventions for matrix ordering.  If `vec` uses the same convention as the underlying matrix storage, than `vec` and `unvec` are zero cost (or the same cost as memory allocation and copying).

I don't say this lightly -- the column-major convention makes more sense mathematically, but it is a convention, and if it buys us performance, maybe we should think about it.

Reply:
Yeah `numpy` defaults to C (row-major) ordering but you ask for Fortran (column-major) storage if you want but you'd be fighting the defaults everywhere.